Providers/Scripts/2.4x-2.5x/Scripts/Tests/dummy_nxOMSAutomationWorker_files/oms_conf_auto_manual.conf

```diff
@@ -0,0 +1,10 @@
+[oms-worker-conf]
+auto_registered_worker_conf_path = /home/bhbrahma/PowerShell-DSC-for-Linux/Providers/Scripts/2.6x-2.7x/Scripts/Tests/temp/worker.conf
```

Comment:
dummy file and path is inconsequential, its just there to check whether the conf file is created with the path we give it.